environments/dev-env/stages/poc2023q1-stage1.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: test.kcp.io/v1alpha1
+kind: Playground
+spec:
+  shards:
+    - name: main
+      workspaces:
+        - name: root
+          workspaces:
+            - name: imw-1
+              others:
+                - name: sync-target-f
+                  source:
+                    file:
+                      path: test/kubectl-kcp-playground/examples/kcp-edge/sync-target-f.yaml
+                - name: sync-target-g
+                  source:
+                    file:
+                      path: test/kubectl-kcp-playground/examples/kcp-edge/sync-target-g.yaml
+              Locations:
+                - name: location-f
+                  labels:
+                    env: prod
+                  instanceSelector:
+                    matchLabels:
+                      example: "yes"
+                      extended: "no"
+                - name: location-g
+                  labels:
+                    env: prod
+                    extended: "yes"
```

Comment:
Looks like the labels on `location-g` prevent `location-g` to be selected by `edge-placement-s`, which is differrent with [our designed scenario](https://github.com/kcp-dev/edge-mc/blob/9a17ccbec614156a58f24efbd436e37ff9b5d115/docs/content/en/docs/Coding%20Milestones/PoC2023q1/example1.md?plain=1#L473).

Reply:
You're right! Just fixed it